File: 1.Domain/Domain.Model/ReaddressStatus.cs
using Core;
using Domain.Model.Enum;

namespace Domain.Model
{
    public class ReaddressStatus : Entity<int>
    {
        public StageStatus FromStatus { get; set; }
        public StageStatus ToStatus { get; set; }
        public int? ReaddressReasonId { get; set; }
        public ReaddressReason ReaddressReason { get; set; }
        public string Feedback { get; set; }
    }
}

File: src/entities/ReaddressStatus.ts
import { StageStatusEnum } from "./enums/stage-status.enum";
import { ProcessCurrentStageEnum } from "./enums/process-current-stage";

export class ReaddressStatus {
  id: number;
  fromStatus: StageStatusEnum;
  toStatus: StageStatusEnum;
  readdressReasonId: number;
  feedback: string;
}